wit/ephemeral/redis-types.wit

```diff
@@ -7,10 +7,17 @@ enum error {
 // The message payload.
 type payload = list<u8>
 
-// An argument and return type for the general-purpose `execute` function.
-variant value {
+// A parameter type for the general-purpose `execute` function.
+variant redis-parameter {
+    str(string),
```

Comment:
Is this useful down at this level? Any difference with encoding strings into `binary`?

Reply:
I figured it would be more ergonomic for many/most languages.  So yes, it's unnecessary, but still seems useful.

Reply:
I guess I've been looking at WIT as _mostly_ a low-level interface for libraries to wrap, though we haven't always done that...

Reply:
If we keep it, this should document that the host must utf-8 encode.

Reply:
I would keep the int type since it enforces redis's actual int range and has the separate wire encoding. Also presumably generated JS bindings will have to solve the JS number precision problem for `int64` _somehow_, so redis bindings over there wouldn't have to invent their own solution.